app/controllers/streams_controller.rb

```diff
@@ -84,6 +85,26 @@ def current_song
     end
   end
 
+  def subscription
+    hijack do |sock|
+      redis_thread = Thread.new do
+        Redis.new.subscribe "songs" do |on|
+          on.message do |channel, song|
```

Comment:
Unused block argument - `channel`. If it's necessary, use `_` or `_channel` as an argument name to indicate that it won't be used.